packages/material-ui-styled-engine-sc/src/GlobalStyles/GlobalStyles.test.js

```diff
@@ -101,4 +101,12 @@ describe('GlobalStyles', () => {
       color: 'rgb(255, 0, 0)',
     });
   });
+
+  it('should raise console.error()', function test() {
+    // if (/jsdom/.test(window.navigator.userAgent)) this.skip();
+
+    expect(() => {
+      styled('span')({ color: undefined });
+    }).toErrorDev('empty color');
```

Comment:
That wouldn't tell me anything I were to use `styled`. Can we expan this a bit to provide some context where this error originated and what the dev should do to avoid it?

Reply:
Yeah sure @eps1lon, I'll change the message, but should uncomment

```js
// if (/jsdom/.test(window.navigator.userAgent)) this.skip();
```

for the test?

Reply:
Looks like we can remove it.